tests/config/driverconfig/powerstore/v2.7.0/driver-config-params.yaml

```diff
@@ -18,7 +18,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: <DriverDefaultReleaseName>-config-params
-  namespace: <DriverDefaultReleaseNamespace>
+  namespace: <DriverDefaultReleaseName>
```

Comment:
Should this not be DriverDefaultReleaseNamespace?

Reply:
Ran  _make driver-unit-test_
![image](https://github.com/dell/csm-operator/assets/109620911/99404f5c-6339-4ea9-bc99-b88a41197b13)

Ran _make unit_test_
![image](https://github.com/dell/csm-operator/assets/109620911/a5b7df44-fb68-4cb3-85ea-a3d64094c7c5)